com.playeveryware.eos/Runtime/Core/Config/JsonConverter/GuidConverter.cs

```diff
@@ -0,0 +1,70 @@
+/*
+* Copyright (c) 2024 PlayEveryWare
+*
+* Permission is hereby granted, free of charge, to any person obtaining a copy
+* of this software and associated documentation files (the "Software"), to deal
+* in the Software without restriction, including without limitation the rights
+* to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+* copies of the Software, and to permit persons to whom the Software is
+* furnished to do so, subject to the following conditions:
+*
+* The above copyright notice and this permission notice shall be included in
+* all copies or substantial portions of the Software.
+*
+* THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+* IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+* FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+* AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+* LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+* OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+* SOFTWARE.
+*/
+
+namespace PlayEveryWare.EpicOnlineServices
+{
+    using Newtonsoft.Json;
+    using System;
+
+    public class GuidConverter : JsonConverter
+    {
+        public override void WriteJson(JsonWriter writer, object value, JsonSerializer serializer)
+        {
+            if (value is Guid guid)
```

Comment:
Will this work if `value` is `Guid?`, or only if the value type is exactly `Guid`?

Reply:
It's not possible for value to be `Guid?`

Reply:
Perhaps I don't understand this JSON writer well enough, but why is possible to Read JSON where the type is a `Guid?` but not write one?

Reply:
It is possible - it's just that we don't have a need to implement it for `Guid?`

We _could_ add it for future-proofing - but that would be adding functionality that isn't currently needed, which seemed like a bad thing to do. Support for `Guid?` would be easy to add though - if you'd like it added.